<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pclviewer</class>
 <widget class="QMainWindow" name="pclviewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1030</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>pclviewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKWidget" name="qvtkWidget" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>760</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add_cloud">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Cloud</string>
    </property>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>500</y>
      <width>760</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="color_video_label">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>10</y>
      <width>760</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string>Video</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>132</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_remove_cloud">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Cloud No.</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_clouds_serial">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>69</width>
      <height>51</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>NONE</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Save All</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
